lib/iris/coords.py

```diff
@@ -1649,8 +1648,8 @@ def bounds(self):
         """
         if self._bounds is not None:
             bounds = self._bounds
-            if isinstance(bounds, biggus.Array):
-                bounds = bounds.ndarray()
+            if is_lazy_data(bounds):
+                bounds = bounds.compute()
```

Comment:
Anything with "compute" in it should be hit by https://github.com/SciTools/iris/issues/2422 to turn it into an "as_concrete_data" call.